client/src/app/slides/assignments/poll/poll-slide.component.ts

```diff
@@ -10,21 +11,87 @@ import { SlideData } from 'app/core/core-services/projector-data.service';
     styleUrls: ['./poll-slide.component.scss']
 })
 export class PollSlideComponent extends BaseSlideComponent<PollSlideData> {
-    // TODO: Remove the following block, if not needed.
-    // This is just for debugging to get a console statement with all recieved
-    // data from the server
     private _data: SlideData<PollSlideData>;
+
+    public pollValues: SummaryPollKey[] = ['votesno', 'votesabstain', 'votesvalid', 'votesinvalid', 'votescast'];
+
     @Input()
     public set data(data: SlideData<PollSlideData>) {
         this._data = data;
-        console.log('Data: ', data);
+        this.setPercents();
     }
+
     public get data(): SlideData<PollSlideData> {
         return this._data;
     }
-    // UNTIL HERE
 
-    public constructor() {
+    public constructor(private pollService: AssignmentPollService) {
         super();
     }
+
+    public getVoteString(rawValue: string): string {
+        const num = parseFloat(rawValue);
+        if (!isNaN(num)) {
+            return this.pollService.getSpecialLabel(num);
```

Comment:
it would be neat if poll.service.ts could use `toLocaleString(this.translate.currentLang)`, so we have proper decimal separators, but apparently, `TranslateService` is undefined (if put into constructor and used there) when coming from a slide.
Is there a way to get there (basically 'getting the current language setting) @FinnStutzenstein